.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE]"
+labels: feature
+assignees: u11d-michal-miler, u11d-bartlomiej-galezowski
+
+---
+
+**Is your feature request related to a problem? Please describe.**
```

Comment:
I'd change verbs in title to nouns. What do you think?

```suggestion
**Problem description**
```

Reply:
Yes, that is more readable.